components/progress/style/index.tsx

```diff
@@ -217,7 +220,7 @@ const genCircleStyle: GenerateStyle<ProgressToken> = (token: ProgressToken): CSS
 };
 
 const genStepStyle: GenerateStyle<ProgressToken> = (token: ProgressToken): CSSObject => {
-  const { progressCls } = token;
+  const { componentCls:progressCls } = token;
```

Comment:
```suggestion
  const { componentCls: progressCls } = token;
```

提交时没自动跑 husky 么，怎么代码风格会有问题？

Reply:
这个我也觉得很奇怪，为什么提交了还有这样的问题